_data/compose-cli/docker_compose.yaml

```diff
@@ -233,6 +267,7 @@ options:
     Specify an alternate working directory
     (default: the path of the Compose file)
   deprecated: false
```

Comment:
We should update the code to mark this deprecated, that way it's also annotated as such, and shows a `deprecated` badge, like this;

<img width="994" alt="Screenshot 2022-05-02 at 11 27 27" src="https://user-images.githubusercontent.com/1804568/166214377-031b039c-bd07-4d8c-8aa3-c4fdebf2c853.png">


To mark it deprecated, there's two options;

actually deprecate it; this makes the flag hidden on the command-line and produces an error ("deprecated: deprecation message") when someone tries to use it; https://github.com/docker/cli/blob/6c9eb708fa6d17765d71965f90e1c59cea686ee9/cli/command/context/update.go#L54-L55


"soft" deprecate; adding a `deprecated` annotation; the flag remains to be functional, but the annotation is used in our docs to show that it's deprecated. Soft deprecation can be useful in situations where a flag should no longer be used, but where _fully_ removing it would cause too much of a backward-breaking change. In that case, you can also mark the flag as `hidden` in Cobra so that it doesn't show on the command line `--help` output to discourage new users from using it; example; https://github.com/docker/cli/blob/v20.10.14/cli/command/context/create.go#L69